lib/stream_data.ex

```diff
@@ -698,7 +698,7 @@ defmodule StreamData do
 
   ## Examples
 
-  Let's build a generator that returns a binary around 25% of times and a
+  Let's build a generator that returns a binary around 25% of times and an
```

Comment:
*of the time and an integer around 75% of the time.*

Reply:
everything else is good

Reply:
Thanks, I have updated it. :+1: